### DISCUSSION TITLE
Replaying Random Mod score crashes the game

### DISCUSSION THREAD

**Discussion by Theighlin** [ppy/osu#15744] [Q&A] [answered]
As per title (only verified in standard)

**Comment by peppy** [comment]
Do you get enjoyment from making us ask for more information?

**Reply by Theighlin** [reply]
No, I don't, but i didn't see any error in the logs and recording a video of the game crashing didn't seem useful to me, but that's not for me to decide i suppose.
 
Tested on a couple of maps after playing random mod. doesn't seem to happen with other ones altough i haven't tride them all.

lazer 2021.1120.0

https://user-images.githubusercontent.com/53081750/142872910-4a74e49d-022e-4a2f-89d3-ab7929ff1213.mp4

[runtime.log](https://github.com/ppy/osu/files/7581621/runtime.log)

**Reply by peppy** [reply]
Please link the beatmap you can reproduce on. I can't reproduce in local testing.

**Reply by Theighlin** [reply]
could reproduce on these 2 maps

https://osu.ppy.sh/beatmapsets/788905#osu/1655719
https://osu.ppy.sh/beatmapsets/955866#osu/1996783

Ok so it doesn't seem to happen on all maps and could be diff dependant.

https://user-images.githubusercontent.com/53081750/142893879-3c0bc77d-a30e-452c-b1c1-3d6f1101ea00.mp4

**Comment by Loreos7** [comment, marked as answer]
Seems like it happens only with the seeds >1 billion. Here is my investigation:

https://user-images.githubusercontent.com/86934170/142901568-8925c352-562d-4418-9a02-c2db30dd1df7.mp4

**Reply by bdach** [reply]
Thanks, that was helpful. Proposal for fix submitted in #15745.
